04-onion-routing.md

```diff
@@ -263,6 +263,10 @@ It is formatted according to the Type-Length-Value format defined in [BOLT #1](0
     2. data:
         * [`32*byte`:`payment_secret`]
         * [`tu64`:`total_msat`]
+    1. type: 10 (`hold_fee`)
```

Comment:
Doesn't that conflict with onion messages in #759

Reply:
This PR is only conceptual at this stage.